cloudaux/orchestration/aws/iam/managed_policy.py

```diff
@@ -69,5 +70,8 @@ def get_managed_policy(managed_policy, flags=FLAGS.ALL, **conn):
     :param conn:
     :return:
     """
+    if not managed_policy.get('Arn'):
```

Comment:
I'm curious... the tests were working previously without this code added.

Reply:
I found the source: The exception is being raised from here: https://github.com/Netflix-Skunkworks/cloudaux/blob/master/cloudaux/orchestration/aws/__init__.py#L28-L45

However, you are correct that it does not match `message=`.

Reply:
Yeah, the pytest documentation was confusing here which ultimately lead to them deprecated the `message` parameter entirely.  Same exception was being thrown with the  wrong text :-)

Reply:
I'm torn on this code, because I actually added this code, and then my colleague removed it: https://github.com/Netflix-Skunkworks/cloudaux/commit/ace3daf4ce4205a275fe3f7fa119fc4affc6f5c2#diff-598eec90f95397f025f8b86368accc3c

I don't recall if he needed to or not for Managed Policies...

Reply:
Happy to revert this and update the unit test to match the old behaviour. FWIW, the same exception is thrown and I would expect code handling this error condition to not be checking the message associated with the exception but totally understand not wanting to break code elsewhere relying on the changed behaviour.

Reply:
I think this should be fine.